.github/workflows/pr.yaml

```
@@ -17,6 +17,7 @@ jobs:
       - conda-cpp-tests
       - conda-python-build
       - conda-python-tests
+      - docs_build
```

Use dashes to match existing job names.
```suggestion
      - docs-build
```